ktor-features/ktor-locations/jvm/src/io/ktor/locations/LocationsInfo.kt

```diff
@@ -16,7 +16,7 @@ import kotlin.reflect.*
  * @property queryParameters is a list of properties stored in query parameters
  */
 @KtorExperimentalLocationsAPI
-data class LocationInfo internal constructor(
+public data class LocationInfo internal constructor(
     val klass: KClass<*>,
```

Comment:
So it's not required here to provide visibility modifiers? Why? Is it relaxed for internals or for data classes?

Reply:
for data classes